src/main/java/io/split/android/client/SplitFilter.java

```diff
@@ -71,6 +72,9 @@ public static SplitFilter byPrefix(@NonNull List<String> values) {
     }
 
     public static SplitFilter bySet(@NonNull List<String> values) {
+        if (values == null) {
```

Comment:
Just being extra careful since I'm expecting this to be non null downstream and this method can be called by the consumer. The annotation won't prevent using a null value, it just provides a hint.